surf/spring-surf/spring-surf/src/main/java/org/springframework/extensions/surf/site/AuthenticationUtil.java

```diff
@@ -52,15 +54,22 @@ public static void logout(HttpServletRequest request, HttpServletResponse respon
         // remove cookie
         if (response != null)
         {
-            Cookie userCookie = new Cookie(COOKIE_ALFUSER, "");
-            userCookie.setPath(request.getContextPath());
-            userCookie.setMaxAge(0);
-            // MNT-20208 (LM-190131): get "http.secured.session" flag in JAVA_OPTS if available,
-            // and use it to set "secure" and "httpOnly" attributes.
-            boolean securedSession = getHttpSecuredSession();
-            userCookie.setSecure(securedSession);
-            userCookie.setHttpOnly(securedSession);
-            response.addCookie(userCookie);
+            String userCookie = "alfUsername3=; Path=" + request.getContextPath() + "; Max-Age=0;";
+            // MNT-20208 (LM-190131): get "http.secured.session" flag in JAVA_OPTS if available,
+            // and use it to set "secure" and "httpOnly" attributes.
+            boolean securedSession = getHttpSecuredSession();
+            if (securedSession)
+            {
+                userCookie = userCookie + " Secure; HttpOnly;";

```

Comment:
Is it ok to just have " Secure; HttpOnly;" in case of a secured session? Shouldn't we have " Secure=true; HttpOnly=true;" ?
Check out this comment: https://issues.alfresco.com/jira/browse/MNT-20208?focusedCommentId=636762&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-636762